bin/clean-tsv-metadata

```diff
@@ -36,10 +41,15 @@ if __name__ == "__main__":
             type=int,
             help="Number of columns to enforce.\n")
 
+    parser.add_argument(
+            "--sort-col",
+            type=str,
+            help="A column name (or 1-based index for headerless tsvs) to prioritize sorting; if not used, sorting will be alphabetical by columns in the order they appear.`.\n")
+
     parser.add_argument(
```

Comment:
Out of curiosity, which TSV motivated you to add this option?

Reply:
The gisaid annotations so that I could keep paper annotations for the same sequences together.